app/models.py

```diff
@@ -0,0 +1,8 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class Actors:
```

Comment:
We named classes in Python in singular. And also please fix all over the project
```suggestion
class Actor:
```